Runtime/Spectator/SpectatorSettings.cs

```diff
@@ -10,9 +15,61 @@ namespace Innoactive.Creator.UX
     public class SpectatorSettings : SettingsObject<SpectatorSettings>
     {
         private static SpectatorSettings settings;
+        public readonly string KeyBindingsPath = "Resources/KeyBindings";
+        public readonly string KeyBindingsAssetName = "SpectatorKeyBindings.inputactions";
+
+        private void OnEnable()
+        {
+#if ENABLE_INPUT_SYSTEM
+            if (KeyBindingSettings == null && !Application.isPlaying)
+            {
+                KeyBindingSettings = DuplicateAndSetKeyBindings(KeyBindingsAssetName, KeyBindingsPath);
+            }
+#endif
+        }
+
+#if ENABLE_INPUT_SYSTEM
+        /// <summary>
+        /// Duplicates and returns an InputActionAsset in the Resources of the project.
+        /// </summary>
+        /// <param name="originalAssetName">Name of the file to duplicate.</param>
+        /// <param name="targetPath">Destination path.</param>
+        /// <param name="targetAssetName">Optional destination name.</param>
+        /// <returns>The copied InputActionAsset.</returns>
+        public static InputActionAsset DuplicateAndSetKeyBindings(string originalAssetName, string targetPath, string targetAssetName = null)
+        {
+            if (targetAssetName == null)
+            {
+                targetAssetName = originalAssetName;
+            }
+
+            string assetName = originalAssetName.Split('.')[0];
+            InputActionAsset inputActionAsset = Resources.Load<InputActionAsset>(assetName);
+
+            string path = Path.Combine(Application.dataPath, targetPath);
+            string assetPath = Path.Combine("Assets", targetPath, targetAssetName);
+
+            Directory.CreateDirectory(path);
+            if (AssetDatabase.CopyAsset(AssetDatabase.GetAssetPath(inputActionAsset), assetPath))
```

Comment:
These method might not be called on runtime, but `AssetDatabase` is part of `UnityEditor` and won't compile.

Reply:
Yes it does not build~